components/workspacekit/cmd/rings.go

```diff
@@ -917,17 +918,20 @@ func connectToInWorkspaceDaemonService(ctx context.Context) (*inWorkspaceService
 	const socketFN = "/.workspace/daemon.sock"
 
 	t := time.NewTicker(500 * time.Millisecond)
+	errs := errors.New("errors of connect to ws-daemon")
 	defer t.Stop()
 	for {
 		if _, err := os.Stat(socketFN); err == nil {
 			break
+		} else if !os.IsNotExist(err) {
+			errs = fmt.Errorf("%v; %w", errs, err)
```

Comment:
nit
```suggestion
                        errs = fmt.Errorf("%v: %w", errs, err)
```

Reply:
@jenting Thanks for your review. I'm a beginner of go language but please let me why `:` is better than `;`.

Reply:
It's not language specific, it's the coding convention.
Mostly, the error message would be wrap as `xxx: yyy`.